### ISSUE TITLE
About the test data

### ISSUE THREAD

**Issue by jiali1025** [QizhiPei/SMT-DTA#5]
Hi Zhipei,

It is a very nice study! I am just a bit confused about the test set you used for DTI task. Deep affinity includes several generalized test sets, did you include them? Or the model is just tested on the randomly selected test set?

**Comment by jiali1025** [comment]
In addition, since you said "So we directly use the [IC50.tar.xz, Ki.tar.xz](https://github.com/Shen-Lab/DeepAffinity/tree/master/data/dataset) data processed by DeepAffinity in our experiments." in another issue. I am wondering did you use the same train and test tsv data provided by them or you recombine the data file and randomly split it into 6:1:3 as you said in your paper. Since for DeepAffinity, it has train 70% test 30%, and four hold-out datasets.

**Comment by QizhiPei** [comment]
Hi Jiali,

Thanks for your interest in our work. I hope the following answers can help you.

1. Our model is not tested on 4 generalized test sets.
2. The test set(30%) is the same with DeepAffinity for fair comparison, and we additionly extract about 10% from training set as our valid set for hyperparametric tuning.

**Comment by jiali1025** [comment]
Thanks for your answer!
